docs/conf.py

```diff
@@ -19,12 +19,12 @@
 
 project = 'METviewer'
 author = 'UCAR/NCAR'
-author_list = "Burek, T., T. Jensen, I. Srivastava, M. Win-Gildenmeister, L. Goodrich, J. Prestopnik, L. Blank, and T. Fowler"
-version = "3.0.0-rc1"
+author_list = "Jensen, T., I. Srivastava, M. Win-Gildenmeister, L. Goodrich, J. Prestopnik, L. Blank, T. Fowler, and T. Burek"
+version = "3.0.0"
 verinfo = version
 release = f'{version}'
 release_year = '2024'
-release_date = f'{release_year}-11-13'
+release_date = f'{release_year}-12-18'
```

Comment:
```suggestion
release_date = f'{release_year}-12-19'
```
Modified to match the release notes documentation date. @bikegeek if you do plan to release tonight, please modify the date in the release-notes.rst file.